test/studies/shootout/submitted/SysCTypes.chpl

```diff
@@ -0,0 +1,3 @@
+compilerWarning("'SysCTypes' no longer exists... this needs to be resubmitted");
```

Comment:
This is the only thing in this change that looks "odd" to me.  I'm guessing you're putting this as a reminder that this code has changed and we should resubmit it to The Computer Language Benchmarks Game?

Reply:
That's right.  Basically, we try to keep the `studies/shootout/submitted/` directory (a) in sync with our submitted versions and (b) running so that we know if we make changes that improve or hurt their performance between releases.  I should've submitted an update to this test that used `CTypes` when I was having the site update to 1.26.0 but didn't for some reason.  So this keeps things running while serving as a reminder to me when I do the post-1.27 sync with the site that I need to do something about it (one of my release responsibilities).